src/app/components/site/Header.tsx

```diff
@@ -29,31 +27,23 @@ export const Header = () => {
               <div className="header-links ml-auto pr-3 px-md-3 d-flex align-items-center d-print-none">
                 {user &&
                   (!user.loggedIn ? (
-                    <React.Fragment>
-                      <div className="login mx-5 mx-sm-2">
-                        <Link to="/login">
-                          <span>Log in</span>
-                        </Link>
-                      </div>
-                      <div className="signup m-0 mr-md-4 ml-md-3">
-                        <Link to="/register">
-                          <span>Sign up</span>
-                        </Link>
-                      </div>
-                    </React.Fragment>
+                    <div className="login mx-4 mx-sm-2 d-flex flex-row">
+                      <Link to="/login" className="mr-3">
+                        <span>Log in</span>
+                      </Link>
+                      <Link to="/register">
+                        <span>Sign up</span>
+                      </Link>
+                    </div>
                   ) : (
-                    <React.Fragment>
-                      <div className="my-account mx-5 mx-sm-2">
-                        <Link to="/account">
-                          <span>{`${!["xs"].includes(deviceSize) ? "My " : ""}Account`}</span>
-                        </Link>
-                      </div>
-                      <div className="logout m-0 mr-md-4 ml-md-3">
-                        <Link to="/logout">
-                          <span>Logout</span>
-                        </Link>
-                      </div>
-                    </React.Fragment>
+                    <div className="my-account mx-5 mx-sm-2 d-md-flex flex-md-row">
```

Comment:
just to check if this is aligned the same as the signed out state?

Reply:
Originally I thought there was not enough space. However with the changes above the space is there, just about. 
So I have made it to be the same styling .